docs/tutorials/send_receive_ingress.md

```diff
@@ -16,33 +16,97 @@ Before you start, you need to have access to a running Kubernetes cluster
+You can view the list of minikube maintainers at: https://github.com/kubernetes/minikube/blob/master/OWNERS
+  - Using image registry.k8s.io/ingress-nginx/kube-webhook-certgen:v20231011-8b53cabe0
+  - Using image registry.k8s.io/ingress-nginx/kube-webhook-certgen:v20231011-8b53cabe0
+  - Using image registry.k8s.io/ingress-nginx/controller:v1.9.4
+* Verifying ingress addon...
+* The 'ingress' addon is enabled
+
 $ minikube kubectl -- patch deployment -n ingress-nginx ingress-nginx-controller --type='json' -p='[{"op": "add", "path": "/spec/template/spec/containers/0/args/-", "value":"--enable-ssl-passthrough"}]'
+
+deployment.apps/ingress-nginx-controller patched
 ```
+
+#### Make sure the domain of your cluster is resolvable
+
+If you are running your OpenShift cluster locally, you might not be able to
+resolve the urls to IPs out of the blue. Follow [this guide]({{< ref
+"/docs/help/hostname_resolution" >}} "set up dnsmasq") to configure your setup.
+
+This tutorial will follow the simple /etc/hosts approach, but feel free to use
+the most appropriate one for you.
+
 ### Deploy the operator
 
 #### create the namespace
 
-```console
+```{"stage":"init"}
 $ kubectl create namespace send-receive-project
+
+namespace/send-receive-project created
+
 $ kubectl config set-context --current --namespace=send-receive-project
+
+Context "minikube" modified.
 ```
 
 Go to the root of the operator repo and install it:
 
-```console
-$ cd send-receive-project
+```{"stage":"init", "rootdir":"$operator"}
 $ ./deploy/install_opr.sh
+
+Deploying operator to watch single namespace
+Client Version: 4.15.0-0.okd-2024-01-27-070424
+Kustomize Version: v5.0.4-0.20230601165947-6ce0bf390ce3
+Kubernetes Version: v1.28.3
+customresourcedefinition.apiextensions.k8s.io/activemqartemises.broker.amq.io created
+customresourcedefinition.apiextensions.k8s.io/activemqartemisaddresses.broker.amq.io created
+customresourcedefinition.apiextensions.k8s.io/activemqartemisscaledowns.broker.amq.io created
+customresourcedefinition.apiextensions.k8s.io/activemqartemissecurities.broker.amq.io created
+serviceaccount/activemq-artemis-controller-manager created
+role.rbac.authorization.k8s.io/activemq-artemis-operator-role created
+rolebinding.rbac.authorization.k8s.io/activemq-artemis-operator-rolebinding created
+role.rbac.authorization.k8s.io/activemq-artemis-leader-election-role created
+rolebinding.rbac.authorization.k8s.io/activemq-artemis-leader-election-rolebinding created
+deployment.apps/activemq-artemis-controller-manager created
 ```
 
 Wait for the Operator to start (status: `running`).
-```console
-$ kubectl get pod --namespace send-receive-project --watch
-```
 
+```{"stage":"init", "runtime":"bash", "label":"wait for the operator to be running"}
+$ kubectl wait pod --for=condition=Ready --namespace=send-receive-project $(kubectl get pods --namespace send-receive-project | awk '/activemq-artemis-controller-manager/ {print $1;exit}')
```

Comment:
That's shorter and simpler to read, let me try this out

Reply:
I think that in the end, it might just make sense to wait for all the pods of the send-receive-project namespace, since we've just created it, there's no risk of having to wait to other resources.

Reply:
I've done just that, is that good for you @brusdev ?